src/components/Tokens/state.ts

```diff
@@ -13,6 +13,7 @@ export const filterNetworkAtom = atom<SupportedChainId>(SupportedChainId.MAINNET
 export const filterTimeAtom = atom<TimePeriod>(TimePeriod.DAY)
 export const sortCategoryAtom = atom<Category>(Category.marketCap)
 export const sortDirectionAtom = atom<SortDirection>(SortDirection.decreasing)
+export const tokensBannerDismissedAtom = atomWithStorage<boolean>('tokensBannerDismissed', false)
```

Comment:
this should live in App level state... tbh would recommend using redux for this, the user slice